app/controllers/api/health.php

```diff
@@ -388,6 +389,33 @@
         $response->dynamic(new Document([ 'size' => $size ]), Response::MODEL_HEALTH_QUEUE);
     }, ['response']);
 
+App::get('/v1/health/certificate')
+    ->desc('Get status of certificate for a domain to check whether it is still valid or expired.')
+    ->groups(['api', 'health'])
+    ->label('scope', 'health.read')
+    ->label('sdk.auth', [APP_AUTH_TYPE_KEY])
+    ->label('sdk.namespace', 'health')
+    ->label('sdk.method', 'getCertificate')
+    ->label('sdk.description', '/docs/references/health/get-certificate.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_HEALTH_STATUS)
+    ->param('domain', null, new Domain(), 'Domain name')
+    ->inject('response')
+    ->action(function (string $domain, Response $response) {
```

Comment:
Let's add a permission check here.
If I provide API key of my project which has a custom domain added, then the health check should run. but if I try to validate domain which doesn't belong to my project, it should give 404